pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2532,9 +2549,37 @@
 ::  +load: migrate old state to new state (called on vane reload)
 ::
 ++  load
-  |=  old=axle
+  =>  |%
+    ++  axle-old
+      %+  cork
+        axle
+      |=  =axle
+      axle(date %~2020.10.18)
+  --
+  |=  old=$%(axle axle-old)
   ^+  ..^$
-  ..^$(ax old)
+  ::
+  ~!  %loading
+  ?-    -.old
+      %~2020.10.18
+    =/  [* d=duct *]
+      %+  snag  0
+      %+  skim  bindings.server-state.old
```

Comment:
This a fairly wonky way to get the `duct` from the `%init` task, any better ideas?

Reply:
The reason you're getting the duct from the `%init` task is... so that the binding is exactly equal between newly booted ships, and ones that went through this upgrade path?

I probably wouldn't worry about this. The way this system is set up right now, using ducts for "permissions", is flawed in many ways. And it's a binding in the reserved namespace (`/~`), so eyre's the only one who can touch it anyway.

Reply:
Yeah that was my idea. 

Do you think we could construct a `duct` here that somehow communicates that it came from an eyre upgrade or just punt and go with `*duct`?

Reply:
Put some prints in and found out it's `~[/dill //term/1]` on a fakezod. I assume that's what it always is so I'll just hardcode that.

Reply:
Ah, yes. It is indeed always that, but it's better to use `outgoing-duct` from the state, which should be equivalent.

Reply:
Ah I see, fixed.